src/words/environment/WordsObject.java

```diff
@@ -13,13 +15,35 @@
 	private Position cell;
 	private String currentMessage;
 	private Action lastAction;
-
+	private HashMap<WordsObject, ArrayList<Property>> referers;
```

Comment:
not sure i follow here. do you mean a Set of WordsObjects, a Set of Propertys, or a Set of a new class that combines the two? I'll comment more on my thinking behind my implementation in the note below about what if an object removes a property that refers to this object.